### PR TITLE
SAK-24201 No error on uploading a grading archive to assignment that exceeds quota

### PR DESCRIPTION
**Pull request by joaquinmarques**

### CODE REVIEW

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 						onclick="ASN.invokeDownloadUrl('$accessPointUrl', '$actionString', '$alertMessage', '$contextString', '$viewString', '$searchString', '$showSubmissionByFilterSearchOnly', this);" value="$tlang.getString('downloadall.button.download')" />
 			#else
 				<input type="button" name="uploadButton" id="uploadButton"  accesskey="s" class="active"
-						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action 'doUpload_all')'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
+						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action 'doUpload_all' 'sakai_csrf_token=$validator.escapeUrl($sakai_csrf_token)')'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
```
Comment (ottenhoff) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: The two apostrophes looks odd to me....

Comment (joaquinmarques) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: I've tested without two apostrophes and empty screen appears and this error is displayed in log:

ERROR [http-nio-8080-exec-4] velocity.sakai.assignment.tool.log Parser Error: /vm/assignment/chef_assignments_instructor_uploadAll.vm
org.apache.velocity.runtime.directive.MacroParseException: Invalid arg #2 in VM #toolLinkParam/vm/assignment/chef_assignments_instructor_uploadAll.vm[line 105, column 164]

Comment (mpellicer) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: Hi @ottenhoff, 

We've tested this and it's working fine, i think the apostrophes are necessary.

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: +1, the apostrophes are actually needed here

**Pull request merged**